docs/installation.md

```diff
@@ -6,6 +6,7 @@ With `wdi5` [being a service to WebdriverIO](https://webdriver.io/docs/wdio-ui5-
 
 - UI5 app running in the browser, accessible via `http(s)://host.ext:port`.
   Recommended tooling for this is either the official [UI5 tooling](https://github.com/SAP/ui5-tooling) (`ui5 serve`) or some standalone http server like [`soerver`](https://github.com/vobu/soerver) or [`http-server`](https://www.npmjs.com/package/http-server).
+- WDI5 requires at least following UI5 libraries: `sap.base`, `sap.ui.test`
```

Comment:
please describe how the end user is able to install these 2 libs locally (ui5 use?) or any other means how to make them available (cdn?)
and: lowercase `wdi5` of course :)